### DISCUSSION TITLE
OpenWrt 版 alist v3.37.4 “failed migrate database: database disk image is malformed”

### DISCUSSION THREAD

**Discussion by imDazui** [AlistGo/alist#7270] [Q&A]
ARS2 的 OpenWrt 21.02.3 
用的iStore [v0.1.25-0]版alist网盘v3.37.4
不能运行
日志出现
““failed migrate database: database disk image is malformed””
求助该怎么解决

![截屏2024-09-26 17 25 37](https://github.com/user-attachments/assets/cc371bf4-3b7e-4ad5-b231-a804bd5560b9)

**Comment by kaneki-029** [comment]
可能数据库文件损坏了。可以参考网上的教程修一修看能不能好，记得修之前备份一下原数据库。

**Reply by imDazui** [reply]
感谢回复 👍🏻👍🏻

主要是小白用户不会玩命令行，就很尴尬。哈哈哈

好在播放器支持连阿里云可以替一下，夸克算是挂不上了😭
